### PR TITLE
Add a convenience web starter

### PR DESCRIPTION
**Pull request by geoand**
The starter includes both the jaeger Tracer auto-configuration code
as well as OpenTracing's Spring web auto-configuration

### CODE REVIEW

**Comment by geoand** [comment]
@cmoulliard I named the new web starter module as you suggested (`opentracing-jaeger-spring-web-starter`),  however the naming doesn't match the existing module (`opentracing-spring-jaeger-starter`).

I suggest we change the name of the new module from `opentracing-jaeger-spring-web-starter` to `opentracing-spring-jaeger-web-starter`.

@pavolloffay WDYT?

**Comment by cmoulliard** [comment]
LGTM. Can you have a look please @pavolloffay ?

**Comment by geoand** [comment]
@cmoulliard Thanks for checking. Can you also comment on my thoughts about the naming? I don't particularly like having different naming schemes for the starters...

File: README.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 
+## Dependencies
+
+The `opentracing-spring-jaeger-starter` simply contains the code needed to provide a Jaeger implementation of the OpenTracing's `io.opentracing.Tracer`
+interface.
+
+For a project to be able to actually instrument a Spring stack, one or more of the purpose built starters (like `io.opentracing.contrib:opentracing-spring-web-starter` or `io.opentracing.contrib:opentracing-spring-cloud-starter`)  
+would also have to be included in the POM.
+
+The `opentracing-jaeger-spring-web-starter` starter is convenience starter that includes both `opentracing-spring-jaeger-starter` and `opentracing-spring-web-starter`
```
Comment (pavolloffay) [inline README.md]: `opentracing-spring-jaeger-web-starter` seems better to me. All spring related deps starts with `openracing-spring`

**Comment by pavolloffay** [comment]
LGTM just a minor comment for naming conventions

File: README.md (hunk 1 above)
Comment (geoand) [reply README.md]: I also believe what you propose is the appropriate name

Comment (geoand) [reply README.md]: @cmoulliard Do you agree we change the module name to `opentracing-spring-jaeger-web-starter` to fit the naming convention used throughout the other opentracing projects?

Comment (metacosm) [reply README.md]: I agree as well.

Comment (cmoulliard) [reply README.md]: I agree too if we will also follow the same convention to name the following starter `opentracing-spring-zipkin-starter` if we will create it in the future too

Comment (geoand) [reply README.md]: @cmoulliard Yes, that's the plan :wink:

**Pull request merged**